plugins/ffmpeg/FFMPEGWriter.cpp

```diff
@@ -179,7 +179,7 @@ FFMPEGWriter::FFMPEGWriter(const std::string &filename, DeviceSpecs specs, Conta
 	m_input_samples(0),
 	m_deinterleave(false)
 {
-	static const char* formats[] = { nullptr, "ac3", "flac", "matroska", "mp2", "mp3", "ogg", "wav" };
+	static const char* formats[] = { nullptr, "ac3", "flac", "matroska", "mp2", "mp3", "ogg", "wav", "adts" };
```

Comment:
Why is it adts here and not aac like everywhere else?

Reply:
Ah well, apparently the muxer is called adts, so this is fine.

Reply:
Initially my wild guess was also "aac" until I found out that does not actually work... yes ffmpeg calls the muxer "adts".